plugin/evm/validators_normalizer_test.go

```diff
@@ -0,0 +1,88 @@
+package evm
+
+import (
+	"math"
+	"math/rand"
+	"reflect"
+	"testing"
+
+	"github.com/flare-foundation/flare/ids"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
```

Comment:
```suggestion
import (
	"math"
	"math/rand"
	"reflect"
	"testing"

	"github.com/stretchr/testify/assert"
	"github.com/stretchr/testify/require"

	"github.com/flare-foundation/flare/ids"
)
```

Reply:
This is also reorganized
https://github.com/flare-foundation/coreth/pull/21/commits/795a19e455a7a1b0b6a427200d807bfb6ec6d3dc#diff-3bb410a6d1f259ada3678375ee08a83bd2a8bf2e93945e5c92b92a6f9d99e796R11